lib/class/class_fpas_place.dart

```diff
@@ -245,4 +245,52 @@ class FPASPlace extends Place {
       }
     }
   }
+
+  /// Fetch the server settings from the given FPAS server. This sets in the
+  /// preferences the link to the privacy police, the terms of use and
+  /// the faps server url
+  ///
+  /// [newUrl] the url of the new FAPS server
+  ///
+  /// returns true if the data was successfully fetched,
```

Comment:
You should not use booleans to indicate if a function is successful, instead use exceptions. Especially the "or something else went wrong" is exactly why exceptions exist, it allows the caller to properly react to every possible scenario rather than a generic "something went wrong but we don't know what".